packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -51,56 +39,69 @@
   background-color: var(--color-surface);
 }
 
-.triggerIcon {
+.clearSearch {
   display: flex;
-  justify-content: center;
-  align-items: center;
-  width: calc(var(--space-smaller) * 5);
-  height: calc(var(--space-smaller) * 5);
-  border-radius: var(--radius-circle);
-  background-color: var(--color-surface--background);
-}
-
-.search .triggerIcon {
   position: absolute;
+  top: 0;
   top: 50%;
   right: var(--space-small);
   z-index: var(--elevation-tooltip);
-  transform: translateY(-50%);
-}
-
-.optionsList {
-  margin: 0;
+  width: calc(var(--space-smaller) * 5);
+  height: calc(var(--space-smaller) * 5);
   padding: 0;
-  list-style: none;
-}
-
-.optionsList li {
-  padding: var(--space-small) var(--space-base);
+  border: none;
+  border-radius: var(--radius-circle);
+  background-color: var(--color-surface--background);
+  transform: translateY(-50%);
+  justify-content: center;
+  align-items: center;
 }
 
-.selectedOption {
-  background-color: var(--color-surface--background);
+.clearSearch:focus {
+  border: 1px solid var(--color-focus);
```

Comment:
Can you just do `box-shadow: var(--shadow-focus);` and get everything you need here?  You might also need `outline: none;` since it's a button.

Reply:
Okay actually pulled and tested it - `box-shadow` is probably best imo.  `outline: none` isn't needed

Reply:
yeah I wasn't sure if border was the right thing to do here. I should have checked other existing patterns. `box-shadow` looks good to me and I've added it on the other branch